src/components/BugForm/BugForm.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+// firestore
+import { db } from '../../environments/firebase';
+import { doc, setDoc } from "firebase/firestore";
+
+// react icons
+import { AiFillEdit } from 'react-icons/ai';
+
+import './BugForm.scss';
+
+export default function BugForm({ currentUser }) {
+
+    const [report, setReport] = useState();
+
+    const handleReport = (e) => {
+        setReport(e.target.value);
+        console.log(report)
+    }
+
+    const createBugReport = async (e) => {
+        e.preventDefault();
+        try {
+            await setDoc(doc(db, "bugs", currentUser.uid), {
+                bug: report
+            });
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+    return (
+        <div >
+            <form className="bugForm" onSubmit={createBugReport}>
+                <textarea rows={8} cols={20} value={report} onChange={handleReport}></textarea>
+                <button className="iconBtn" type="submit">
+                    Submit Report <AiFillEdit />
+                </button>
+            </form>
+        </div>
+    )
+}
```

Comment:
Okay, what is the purpose of setting a collection of bugs for the user? How should it notify developers of bug? Like manually checking each user's collection? Create a form to send a mail, no need to store bugs in the Firestore.
Create a dropdown or something with types of bug ( unnotified error, notified error, page is not loading search the internet) + textare with placeholder of "please describe the steps to recreate the bug"